dAuthServer/CMakeLists.txt

```diff
@@ -1,4 +1,2 @@
-set(DAUTHSERVER_SOURCES "AuthServer.cpp")
-
-add_executable(AuthServer ${DAUTHSERVER_SOURCES})
+add_executable(AuthServer "AuthServer.cpp")
```

Comment:
This is on theme with how I changed the other server sources the same way so that we can test them.  Would be better to keep it consistent instead of having it be different for this one because it wasn't on theme for a PR.